test/NinjaTools.FluentMockServer.Tests/TestHelpers/XUnitTestBase.cs

```diff
@@ -1,35 +1,93 @@
+using System;
+using System.Text;
 using JetBrains.Annotations;
 using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using Xunit.Abstractions;
 
 namespace NinjaTools.FluentMockServer.Tests.TestHelpers
 {
     public class XUnitTestBase
     {
-        public ITestOutputHelper Output { get; }
-        public ILogger Logger { get; }
-
-        [NotNull]
-        public ILogger<T> CreateLogger<T>() => Output.CreateLogger<T>();
+        public ITestOutputHelper Output { get; protected set; }
+        public ILogger Logger { get; protected set; }
         
         public XUnitTestBase(ITestOutputHelper output)
         {
             Output = output;
             Logger = Output.CreateLogger(GetType().Name);
         }
+
+
+        [NotNull]
+        protected virtual string Dump<T>([CanBeNull] T? expected, [CanBeNull] T? actual) where T : class
+        {
+           var dump = $"{Dump(expected, header: "Expected")}\n{Dump(actual, header: "Actual")}";
+           return dump;
+           // Dump(actual, header: "Actual");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4765163)